utils/learn-lm.json

```diff
@@ -277,6 +277,14 @@
                             "isDir": false,
                             "url": "/learn/ballerina-shell",
                             "id": "ballerina-shell"
+                        },
+                        {
+                            "dirName": "Ballerina CI/CD",
```

Comment:
Instead of having different text for the left nav, can we have the page title itself?